src/Differ/SebastianBergmannDiffer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Differ;
+
+use SebastianBergmann\Diff\Differ;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class SebastianBergmannDiffer implements DifferInterface
+{
+    /**
+     * @var Differ
+     */
+    private $differ;
+
+    public function __construct()
+    {
+        $this->differ = new Differ();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function diff($old, $new)
+    {
+        return implode(
+            PHP_EOL,
```

Comment:
Might be an idea to have this configurable via an optional constructor param, so a windows system can output unix style line endings if desired.